fe/fe-core/src/main/java/com/starrocks/sql/optimizer/MaterializedViewOptimizer.java

```diff
@@ -28,6 +28,11 @@
 public class MaterializedViewOptimizer {
     public MvPlanContext optimize(MaterializedView mv,
                                   ConnectContext connectContext) {
+        return optimize(mv, connectContext, false);
+    }
+    public MvPlanContext optimize(MaterializedView mv,
+                                  ConnectContext connectContext,
+                                  boolean keepView) {
```

Comment:
suggest to use notExpandingView or noInlineView instead of keepView

Reply:
changed to inlineView